infra/modules/monitoring/variables.tf

```diff
@@ -7,3 +7,14 @@ variable "load_balancer_arn_suffix" {
   type        = string
   description = "The ARN suffix for use with CloudWatch Metrics."
 }
+
+variable "email_alerts" {
```

Comment:
variable name could be better. current name makes it sound like a list of alerts.